versions/3.1.0.md

```diff
@@ -2336,6 +2336,20 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+OpenAPI documents are made up of multiple resources. Sometimes resources are all in the same root OpenAPI document (inline resources); also, resources can contain their own documents with `$ref` (separate resources). A schema object could exist as an inline resource, or a separate resource. 
+
+It is important for tooling to be able to detect what meta-schema any given resource wishes to be processed with: JSON Schema Core, JSON Schema Validation, OpenAPI Schema Object, or some custom meta schema.
```

Comment:
I appreciate we're importing the JSON Schema terminology here, but are "resources" only `schemaObjects` or is any component a resource? Only `schemaObjects` can indicate their `$schema` so it is only "important for tooling" when dealing with `schemaObjects`, not for example `parameters` or `responses`.

Reply:
What would be the use-case for specifying JSON Schema Core as the meta-schema for a `schemaObject` / resource?

> Keywords declared in this section, which all begin with "$", make up the JSON Schema Core vocabulary.

A `schemaObject` or resource which only uses `$`-prefixed keywords doesn't sound very useful in an OAS context.

Reply:
This is where it would be good to use the _dialect_ vs _vocabulary_ terminology: https://github.com/json-schema-org/json-schema-spec/issues/902

> JSON Schema Core, JSON Schema Validation

When written out and capitalized like this, these are usually the name of the specification documents, which don't quite map to meta-schemas.  The meta-schema that most people used is referred to as the ["default meta-schema"](https://tools.ietf.org/html/draft-handrews-json-schema-validation-02#section-5) in 2019-09.  We might say "default dialect meta-schema" as we add the "dialect" term in the next draft.

It would be very strange to use only the core vocabulary as all it does is bootstrap the JSON Schema system.  There is no standard meta-schema for the combination of vocabularies in the JSON Schema Core specification document (== core and applicator vocabularies).